GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -177,11 +177,11 @@ protected virtual bool TryRunGVFSWithArgs(string args, out string consoleError)
 
         private bool IsGVFSUpgradeAllowed(out string consoleError)
         {
-            bool isConfirmed = string.Equals(this.CommandToRerun, GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm, StringComparison.OrdinalIgnoreCase);
+            bool isConfirmed = string.Equals(this.CommandToRerun, GVFSPlatform.Instance.Constants.UpgradeConfirmCommand, StringComparison.OrdinalIgnoreCase);
             string adviceText = null;
             if (!this.IsElevated())
             {
-                adviceText = isConfirmed ? $"Run {this.CommandToRerun} again from an elevated command prompt." : $"To install, run {GVFSConstants.UpgradeVerbMessages.GVFSUpgradeConfirm} from an elevated command prompt.";
+                adviceText = isConfirmed ? $"Run {this.CommandToRerun} again from an elevated command prompt." : $"To install, run {GVFSPlatform.Instance.Constants.UpgradeConfirmCommand} from an elevated command prompt.";

```

Comment:
Should ` from an elevated command prompt` be `{GVFSPlatform.Instance.Constants.ElevatedCommandPrompt}`?